src/components/@shared/FormInput/InputElement/Credential/index.tsx

```diff
@@ -37,6 +39,14 @@ export default function Credentials(props: InputProps) {
     setValue('')
   }
 
+  const showAddressName = (address: string) => {
+    const addressKey = Object.keys(verifiedAddresses).filter((key) => {
+      return key.toLowerCase() === address.toLowerCase()
+    })
+    const addressName = verifiedAddresses[addressKey[0]]
+    return addressName?.length > 0 ? `${address} (${addressName})` : address
+  }
+
```

Comment:
suggestion(blocking): in the `Account` component we have a simpler implementation that only uses addresses as keys
if something more in depth is required (like with `toLowerCase` we should move this to the `useAddressConfig` hook instead (e.g. adding. a `getVerifiedAddressName` function)